### PR TITLE
setup postgres seeding

### PR DESCRIPTION
**Pull request by jessebeard**
<blockquote class="trello-card"><a href="https://trello.com/c/4P4OJbWI/6-s-find-a-cheatsheet-for-postgres">S - Find a cheatsheet for PostGRES</a></blockquote>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,11 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 ## Development
 default port 3002
 
+**PostGres**
+install and start Postgres, include login information for it per PG* variables in a .env file in root directory per env.sample
+once Postgres is installed run `yarn install` to install necessary packages.
```
Comment (Implausibility) [inline README.md:85]: Why did you switch from npm to yarn midstream?

File: env.sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+PGHOST=localhost
+PGUSER=root
+PGPASSWORD=""
```
Comment (Implausibility) [inline env.sample:3]: Always useful to have sample `.ENV` files.

Add a newline at the end to keep all the code readers happy.

Comment (jessebeard) [reply env.sample:3]: ty

File: README.md (hunk 1 above)
Comment (jessebeard) [reply README.md:85]: I just want to use it a bit for practice. Also, it has a cat as it's icon, and a few of my friends in webdev insist its better.

**Pull request merged**